corehq/apps/consumption/shortcuts.py

```diff
@@ -91,19 +91,23 @@ def _update_or_create_default(domain, amount, default, type, **kwargs):
 
 def build_consumption_dict(domain):
     """
-    Takes raw rows from couch and builds a dict to
-    look up consumption values from.
+    Builds a dict to look up consumption values from.
     """
     SQLDefaultConsumption.objects.filter(domain=domain)
 
```

Comment:
I feel bad adding another comment :grimacing: but I don't think this line does anything. Am I missing something?
```suggestion
```

Reply:
huh, must have been a copy paste error. fixed.